deploy/test.yaml

```diff
@@ -1,228 +1,3 @@
-apiVersion: v1
```

Comment:
Guess this file was for testing and can be removed.

Reply:
Removed